symphony/bdk/gen/rest.py

```diff
@@ -49,8 +49,12 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
             maxsize = configuration.connection_pool_maxsize
 
         ssl_context = ssl.create_default_context(purpose=ssl.Purpose.CLIENT_AUTH, cafile=configuration.ssl_ca_cert)
+        ssl_context.verify_mode = ssl.CERT_REQUIRED
+
         if configuration.cert_file:
-            ssl_context.load_verify_locations(configuration.cert_file)
+            ssl_context.load_cert_chain(
```

Comment:
This was to align again to the mustache template (something I mistakenly modified)